hack/generator/pkg/config/configuration.go

```diff
@@ -62,7 +66,7 @@ func (config *Configuration) WithExportFilters(filters ...*ExportFilter) *Config
 
 // Initialize checks for common errors and initializes structures inside the configuration
 // which need additional setup after json deserialization
-func (config *Configuration) Initialize() error {
+func (config *Configuration) Initialize(myLocation string) error {
```

Comment:
`path` rather than `myLocation`?

Reply:
And add doc comment that `path` can be either local or absolute?

Reply:
Done and refactored to make it internal to `config`, an implementation detail.